test.md

```diff
@@ -1,16 +1,11 @@
-
 # Test File
 
 This file is for testing per-line commenting.
 
-You can do per-line comments that are associated with a commit, or with a pull request.
-
-To comment on an older line, copy a link to that line and then paste the link into a commit comment.
-
-Or use blame to find the commit where the line was added, and add the comment to that commit.
+* You can do per-line comments that are associated with a commit, or with a pull request.
+* To comment on an older line, copy a link to that line and then paste the link into a commit comment.
+* Or use blame to find the commit where the line was added, and add the comment to that commit.
 
 If commenting on a PR, the reviewer can approve or request changes from the PR.  After the submitter makes changes, GitHub shows the reviewer only changes since the most recent review.
 
-Let's add another line.
 
```

Comment:
In https://github.com/tedhudek/sandbox/commit/9504e922ec724ea4d850693d03231398b5fd2bff#diff-b128b7694e4de962e0088bcb1bcb254fR14 there was content here.